design/observerability-2.md

```diff
@@ -0,0 +1,196 @@
+# Observability #2
+
+This document deprecates [the previous version of Observability](observability.md)
```

Comment:
Replace observerability.md instead of creating a new file (you may want to keep some of the resources from the original file in an appendix/resources in the updated file

Reply:
The resources you mentioned were the reason I decided to go for a separate document. Most of the are non-public and out-of-date but nevertheless they might be useful in some contexts. Once I replace the document, we'll probably lose them (I can't imagine someone searching through our git repo to dig them out). 

So let me ask you one more time - shall I replace that document? Or maybe I convinced you and we can leave it as is - as a separate doc?

Reply:
@slaskawi I don't think we will lose them, as long as we don't force push. There's always git history for it.

Reply:
@stianst @slaskawi would make sense to get some inspiration from Kiali? https://kiali.io/documentation/latest/validations/. Just as an example: https://kiali.io/documentation/latest/validations/#_kia0002_more_than_one_selector_less_object_in_the_same_namespace

Reply:
@abstractj Could you please put specific examples inline to this design? The document you linked is pretty big and touches many different things. It's hard for me distill what inspiration you're talking about.

Reply:
I also replaced the `observability.md` document.

Reply:
@slaskawi my suggestion was pretty much regards the structure followed by Kiali, to provide scenarios and how to solve each issue. But I guess you kind of did it here https://github.com/keycloak/keycloak-community/blob/af2de8a2c4ed2cc0b12e3e789379239c8bdba1bd/design/observerability.md#investigating-failures